package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Simple and complete React hooks testing utilities that encourage good testing practices.",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "engines": {
+    "node": ">=12"
+  },
```

Comment:
I figured if we were going to make a breaking change, we should at least have something breaking in there 😅